SteamBundleSitesExtension.js

```diff
@@ -842,12 +878,40 @@ const siteHandlers = {
         GM_addStyle(`
             .SBSE_container { margin-top: 10px;}
             .SBSE_container > textarea { border: 1px solid #CC001D;}
-            .SBSE_container button { background-color: #CC001D; color: white;}
+            .SBSE_container button { background-color: #CC001D; color: white; border-radius: 3px;}
+            .SBSE_BtnExport { background-color: #CC001D; color: white; border-radius: 3px;}
```

Comment:
Same as above, `.SBSE_container button` and '.SBSE_BtnExport' are the same, please put them together

Reply:
Fixed~